src/main/java/pl/stormit/eduquiz/quizcreator/domain/user/Status.java

```diff
@@ -0,0 +1,6 @@
+package pl.stormit.eduquiz.quizcreator.domain.user;
+
+public enum Status {
+    ACTIVE,
+    INACTIVE,
```

Comment:
Maybe we should add status: verified, unverified, blocked - instead of active, inactive.
We have this information in Notion from the last meeting.